repair/repair.cc

```diff
@@ -1014,6 +1015,14 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
     auto id = repair_tracker().next_repair_command();
     rlogger.info("repair[{}]: starting user-requested repair for keyspace {}, repair id {}, options {}", id.uuid, keyspace, id.id, options_map);
 
+    // TODO Deexie add type and finish task. 
+    auto task = seastar::make_shared<repair_task>(task_id{id.uuid});
```

Comment:
Here, if we use `task_manager::task::impl`, this will look like
```
    auto task = seastar::make_shared<task_manager::task>(task_id{id.uuid}, std::make_unique<repair_task>());
```

BTW, repair could use task_id natively (follow up)